app/base/static/assets/js/vis/simple-line-chart.js

```diff
@@ -1,27 +1,30 @@
 class SimpleLineChart {
+    /*
     CHART_WIDTH = 1000;
     CHART_HEIGHT = 600;
-    GAP = 10;
+    */
+    CHART_WIDTH = document.getElementById('charts').offsetWidth - 50;
+    CHART_HEIGHT = window.innerHeight - 120;
+    GAP = 20;
```

Comment:
These 3 magic numbers happen to be the same as in the barchart. Please consider adding them in `common.js` with a comment why these are selected. I assume these will be reused over and over again in other plots.

Even better, define a function in `common.js` that computes chart dimensions with chart ID as input and reuse the function.

Reply:
get rid of 50 and 120, instead of 
    CHART_WIDTH = document.getElementById('charts').offsetWidth;
    CHART_HEIGHT = window.innerHeight - Common.MainContent_Gap;
, where   Common.MainContent_Gap is the recalculated number stored in common.js

Reply:
have you considered?
> Even better, define a function in common.js that computes chart dimensions with chart ID as input and reuse the function.

it's because the same 3 lines will be reused in many places